cfn-resources/stream-connection/test/stream-connection-sample-cfn-request.json

```diff
@@ -0,0 +1,15 @@
+{
+  "desiredResourceState": {
+    "Profile": "maastha-cloud-dev",
```

Comment:
should we update this profile name to `default`?

Reply:
yess! I will update

Reply:
updated